WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
Wrong.

Zoroastrian Armenians should be allowed to create their culture's Empire. In actuality, there should be no requirements at all, and "AI = NO" should not be in this file.